c10/core/DispatchKey.h

```diff
@@ -59,6 +59,8 @@ enum class DispatchKey : uint8_t {
 
   VariableTensorId,
 
+  ProfilerId,
```

Comment:
Do we want this after autograd? Afaik, the old profiling was before autograd

Reply:
I'm a bit confused about the ordering here. I was under the impression that this is a higher bit index since it's later than the enum. Above it says higher bit indexes are dispatched first because of the leading zeros thing

Reply:
This is the correct position. Lower in the enum = happens earlier / higher priority

Reply:
We could, although it would impact the code we generate (as we now have to bitshift in the other direction)